core/src/ledger/parameters/mod.rs

```diff
@@ -111,6 +115,16 @@ impl Parameters {
             pos_inflation_amount,
         } = self;
 
+        // write max proposal bytes parameter
```

Comment:
We should add validation somewhere that the input isn't insane. Here perhaps?

Reply:
Or maybe it' crazy to validate genesis input. But we will need some mechanism for validation if it is changed by governance. Tomas mentioned something about guarding it with VPs. Maybe a separate pr.

Reply:
we're grabbing this parameter from the toml config, whose deserialization already has some logic that checks if it is in the range [1 B, 90 MiB]

Reply:
I'm not sure if that's enough?

Reply:
Ah, I missed that. So good enough for now. For governance, I'm not so sure as it doesn't require a hard fork but just a change in storage. So we might need this VP idea. But that can be a separate PR.